File: app/src/main/java/mx/edu/upqroo/kristenandroid/services/kristen/containers/Contacto.java
package mx.edu.upqroo.kristenandroid.services.kristen.containers;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Contacto {
    @SerializedName("idContacto")
    @Expose
    private String idContacto;

    @SerializedName("nombre")
    @Expose
    private String nombre;

    @SerializedName("correo")
    @Expose
    private String correo;

    public Contacto() {
    }

    public Contacto(String idContacto, String nombre, String correo) {
        this.idContacto = idContacto;
        this.nombre = nombre;
        this.correo = correo;
    }

    public String getIdContacto() {
        return idContacto;
    }

    public void setIdContacto(String idContacto) {
        this.idContacto = idContacto;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }
}

File: app/src/main/java/mx/edu/upqroo/kristenandroid/api/kristen/containers/Contacto.kt
package mx.edu.upqroo.kristenandroid.api.kristen.containers

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Contacto {
    @SerializedName("idContacto")
    @Expose
    var idContacto: String? = null

    @SerializedName("nombre")
    @Expose
    var nombre: String? = null

    @SerializedName("correo")
    @Expose
    var correo: String? = null
}
